src/com/palmergames/bukkit/towny/object/WorldCoord.java

```diff
@@ -339,4 +339,17 @@ public List<WorldCoord> getCardinallyAdjacentWorldCoords(boolean... includeOrdin
 		}
 		return list;
 	}
+
+	/**
+	 * Receives a Location, and calculates the distance from the centre of the
+	 * WorldCoord (at the same Y level as the given Location,) to the given
+	 * Location.
+	 * 
+	 * @param loc Location to measure distance from.
+	 * @return distance from the location to the centre of the WorldCoord on the same Y level.
+	 */
+	public int getNormalizedDistanceFromLocation(Location loc) {

```

Comment:
I feel that distanceTo (or just distance similar to the Location method) might read slightly better here.

Reply:
This code was removed.